<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Shamima" ignored="0" name="Chrome" tests="6" failures="1" timestamp="2020-09-30T18:40:17 EDT" time="59.012" errors="0">
  <testcase name="verifyLogin" time="9.637" classname="com.osa.tetes.HomePage"/>
  <testcase name="verifyLogin" time="8.902" classname="com.osa.tetes.HomePage"/>
  <testcase name="verifyLogin" time="9.414" classname="com.osa.tetes.HomePage">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at com.osa.tetes.HomePage.verifyLogin(HomePage.java:27)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyLogin -->
  <testcase name="verifyLogin" time="9.891" classname="com.osa.tetes.HomePage"/>
  <testcase name="verifyLogin" time="10.919" classname="com.osa.tetes.HomePage"/>
  <testcase name="verifyLogin" time="9.233" classname="com.osa.tetes.HomePage"/>
</testsuite> <!-- Chrome -->
